packages/bitcore-node/src/models/transaction.ts

```diff
@@ -673,29 +673,33 @@ export class TransactionModel extends BaseTransaction<IBtcTransaction> {
   async _invalidateTx(params: {
     chain: string;
     network: string;
-    invalidTxid: string,
-    replacedByTxid?: string // only provided at the beginning of the ancestral tree. Txs that spend unconfirmed outputs aren't "replaced"
-    invalidParentTxids?: string[] // empty at the beginning of the ancestral tree. 
+    invalidTxid: string;
+    replacedByTxid?: string; // only provided at the beginning of the ancestral tree. Txs that spend unconfirmed outputs aren't "replaced"
+    invalidParentTxids?: string[]; // empty at the beginning of the ancestral tree.
+    simple?: boolean; // if true, don't invalidate descendants
```

Comment:
This param name sticks out to me as one that could use some work. What about 'skipInvalidateDescendants'?